### ISSUE TITLE
The problem about render

### ISSUE THREAD

**Issue by GoogleCodeExporter** [JustLoy77/ebookdroid#154]
```
What steps will reproduce the problem?
1.Let open a pdf and scroll the dividing line of pages to center of the 
device's  
  height.Assume the title show "(12/20)xxxxxx" ,then close the pdf .

2.Open the pdf that I closed again, the bottom part that half of the 11 will   
  show black.
3.

What is the expected output? What do you see instead?


What version of the product are you using? On what operating system?
   the latest version

What device are you using?
  version 2.2

Please provide any additional information below.
```

Original issue reported on code.google.com by `WangNin...@gmail.com` on 17 Feb 2012 at 2:34

**Comment by GoogleCodeExporter** [comment]
```
> the latest version
Which one exactly?
```

Original comment by `Alexander.V.Kasatkin@gmail.com` on 17 Feb 2012 at 7:43

**Comment by GoogleCodeExporter** [comment]
```
Sorry , ebook version 1.4.0
```

Original comment by `WangNin...@gmail.com` on 17 Feb 2012 at 8:50

**Comment by GoogleCodeExporter** [comment]
```
Pls try the r949
```

Original comment by `Alexander.V.Kasatkin@gmail.com` on 17 Feb 2012 at 2:59
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 1.4.0
```

Original comment by `Alexander.V.Kasatkin@gmail.com` on 4 Apr 2012 at 10:44
- Changed state: **Fixed**
